src/components/Dashboard/StudentData/index.js

```diff
@@ -389,7 +389,7 @@ function StudentData() {
                 </button>
               </th>
               <th>
-                Classes Attended
+                Classes Enrolled to
```

Comment:
I may be using my American English bias with this suggestion, but "Classes enrolled in" is how I'd write this.  (I think I make mistakes in Hindi because I sometimes mix up postposition use because of these differences.)  However, since you generally try to avoid ending a phrase/sentence with a preposition, "Enrolled Classes" sounds the best to me.
```suggestion
                Enrolled Classes
```